Kconfig

```diff
@@ -102,4 +102,10 @@ config DOM_CFG_DOMU1_DTB_BIN_FILE
 
 endif #DOM_CFG_BUILTIN_IMAGES
 
+config VCHAN_SHELL_CMDS
```

Comment:
Adding config option must be described in the commit message

Reply:
Fixed